rpc/wrtc_client_stream.go

```diff
@@ -83,8 +84,7 @@ func newWebRTCClientStream(
 //
 // It is safe to have a goroutine calling SendMsg and another goroutine
 // calling RecvMsg on the same stream at the same time, but it is not safe
-// to call SendMsg on the same stream in different goroutines. It is also
-// not safe to call CloseSend concurrently with SendMsg or resetStream.
+// to call SendMsg on the same stream in different goroutines.
```

Comment:
users should get proper errors now if they do things concurrently and in a bad way